config/routes.rb

```diff
@@ -1,29 +1,36 @@
 Rails.application.routes.draw do
   filter :locale
 
-  root 'home#index'
+  root 'main#index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.